src/medal_widget/views.cljs

```diff
@@ -1,12 +1,43 @@
 (ns medal-widget.views
   (:require
    [re-frame.core :as re-frame]
+   [medal-widget.events :as events]
    [medal-widget.subs :as subs]
    ))
 
+(def columns ["gold" "silver" "bronze" "Total"])
+
+(defn table-body [idx {:keys [code gold silver bronze total]}]
+  [:tr {:key code} 
+   [:td (+ idx 1)]
+   [:td code]
+   [:td gold]
+   [:td silver]
+   [:td bronze]
+   [:td total]])
+ 
+
 (defn main-panel []
-  (let [name (re-frame/subscribe [::subs/name])]
+  (let [name (re-frame/subscribe [::subs/name])
+        medals-data @(re-frame/subscribe [::subs/medals-data])]
     [:div
-     [:h1
-      "Hello from " @name]
-     ]))
+     [:h1 "Medals Widget"]
+     [:br]
+     [:div
+      [:table#medals
+       [:thead
+        [:tr
+         [:th "S.no"]
+         [:th "Country"]
+         (map-indexed (fn [idx item]
+                        (println idx)
+                        [:th {:key item} [:button {:class (if (= item "gold")
```

Comment:
It doesn't look like you are marking the selected nav item. You should have a subscription ::subs/selected? which takes one argument `medal` which then checks if the `(= (:selected-sort-option db) medal)`